src/__tests__/inventory/AddInventoryItem.test.tsx

```diff
@@ -1,9 +1,98 @@
+import React from 'react';
+
+import createMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+import { fireEvent, render, screen, waitFor } from '@testing-library/react'
+import { Router, Route } from 'react-router-dom'
+
+import Inventory from '../../inventory/Inventory';
+import { TitleProvider } from '../../page-header/title/TitleContext';
+import { RootState } from '../../shared/store'
+import Permissions from '../../shared/model/Permissions'
+import { createMemoryHistory } from 'history'
+import { Provider } from 'react-redux';
+
+const mockStore = createMockStore<RootState, any>([thunk])
+
+const setup = (
+  route: string,
+  permissions: Permissions[],
+  path: string
+) => {
+  const history = createMemoryHistory({ initialEntries: [route] })
+  const store = mockStore({ user: { permissions } } as any)
+  return {
+    history, ...render(
+      <Provider store={store}>
+        <Router history={history}>
+          <Route path={path}>
+            <TitleProvider>
+              <Inventory />
+            </TitleProvider>
+          </Route>
+        </Router>
+      </Provider>
+    )
+  }
+}
+
 describe('AddInventoryItem', () => {
-  it('add item and cancel buttons render', () => {})
+  it('add item and cancel buttons render', () => {
+    /* 1. Render the component 
+     * 2. Check if the add item button has been rendered
+     * 3. Then, check if the cancel item button renders
+     */
+
+    setup('/inventory/new', [Permissions.AddItem], '/');
+    let buttons = screen.getAllByRole('button');
+    expect(buttons[0].textContent).toEqual(' inventory.actions.add ');
+    expect(buttons[1].textContent).toEqual(' actions.cancel ');
```

Comment:
Interestingly, the RTL-ish way to do this is by searching each button individually using its accessible name, so something like
```
expect(screen.getByRole('button', name: {/inventory.actions.add/i}).toBeInTheDocument()
expect(screen.getByRole('button', name: {/actions.cancel/i}).toBeInTheDocument()
```
Nonetheless, your implementation should be totally fine. The only way this would cause problems is if someone reordered the buttons and 'inventory.actions.add' no longer appeared in the DOM before 'actions.cancel'. This comment also applies to all of the other lines with similar code (i.e. searching for all of the elements at once and then fetching them by index).